blockly-rtc/server/http_server.js

```diff
@@ -34,6 +34,12 @@ http.createServer(async (req, res) => {
     await queryEventsHandler_(res, parsedUrl.query.serverId);
   } else if (req.method === 'POST' && parsedUrl.pathname === '/api/events/add') {
     await addEventsHandler_(req, res);
+  } else if (req.method === 'GET' && parsedUrl.pathname === '/api/markers/query') {
```

Comment:
I'd recommend calling this clients. What you're actually requesting is information about the active clients, which includes their location. Markers are the UI element that represents the client. So /api/clients/query

Reply:
This is a valid point. In reality what we are passing is information about the location of a user. For this change, I updated the API paths.

In a following change I will rename MarkerUpdates to something more along the lines of ClientLocation and also update the names of the handlers in the APIs and server.

I will also switch the language I use from client to user in a subsequent change.